wiki/Multi/en.md

```diff
@@ -0,0 +1,487 @@
+
+### Match Settings
+
+<!-- Right -->
+On the right side, it shows the key settings of the Match Setup.
+Do note that only the host can change these settings except for Mods when Free Mods was enabled.
+
+#### Game Name and Change Password
+
+Starting from the top is the Game Name, which is self-explanatory.
+The host can change the name or create/change the Match Setup's password by pressing the `Change Password` button.
+
+#### Beatmap
+
+![host song change][Multi_host_song_change image]
+
+_When the Host is changing songs, this bar will become a temporary placeholder."_
+
+![host song selection][Multi_SS image]
+
+_Example of how the Song Selection will look like for host._
+
+The middle section is the Beatmap section.
+This section shows the beatmap to be played for this match.
+
+It will show the image, game mode symbol, the title, artist of the song, beatmapper, difficulty selected and the star rating of the difficulty.
+
+![Information about the beatmap when hovered over][Multi_beatmapinfo image]
+
+_When hovered, it will show BPM, Length, Last Played, CS, AR, OD, HP, and Stars._
+
+If the player does not have the beatmap, some information will be shown, and one of these will occur:-
+
+Beatmap status | Effect
+---|---
+**Ranked/Approved/Pending/Graveyard** | The player will be directed to the **beatmap page by player's default web browser**. If the player has **osu!direct**, it will be **downloaded and extracted automatically upon click**.
+**Not uploaded or not up-to-date**    | It will **not link to the beatmap**. The host should find the link (in the Beatmap Listing) for the player if the issue persists.
+**Cannot update the beatmap**         | The [host is probably playing a modified version.][modified version link]
+
+#### Mods
+
+![Host mods view][Multi_mods_host image]
+
+_Host game mode view_
+
+![Player mods view][Multi_mods_player image]
+
+_Player game mode view (Free Mode ON)_
+
+The lower left section is the Mods section.
+This section will show the [game mods][Game Modifiers wikilink] used for this match.
+For host-side, the host can enable Free Mods to allow players to use any mods except Speed ([Double Time (DT)][Double Time wikilink]/[Nightcore (NC)][Nightcore wikilink] or [Half Time (HT)][Half Time wikilink]) mods.
+Host-selected mods other than the Speed mods will not be enforced upon players.
+
+#### Team Mode, Win Condition, Tag Colour
+
+The lower right section will shows how the beatmap will be played in the match (Team Mode) and the Win Condition.
+
+##### Team Mode
+
+Team Mode contains four ways of how a match will be played:-
+
+Team Mode | Description
+---|---
+**Head to Head** | Compete against each other to reach the top spot of the match leaderboard.
+**Team Vs**      | Compete against Red/Blue team for match supremacy.
+**Tag Coop [osu! only, UNRANKED]**    | Team up to complete the beatmap, one combo at a time together.
```

Comment:
I'd not bold the items in the square brackets *unless they are apart of the drop down list names of modes